### PR TITLE
Wip/bl/ad3552r qspi axi

### PR DESCRIPTION
**Pull request by spectrum70**
## Pull Request Description

Please replace this with a detailed description and motivation of the changes. 
You can tick the checkboxes below with an 'x' between square brackets or just check them after publishing the PR. 
If this PR contains a breaking change, list dependent PRs and try to push all related PRs at the same time.

## PR Type
- [x] Bug fix (change that fixes an issue)
- [x] New feature (change that adds new functionality)
- [ ] Breaking change (has dependencies in other repos or will cause CI to fail)

## PR Checklist
- [x] I have followed the [Coding style guidelines](http://analogdevicesinc.github.io/no-OS/drivers_guide.html#coding-style)
- [x] I have performed a self-review of the changes
- [x] I have commented my code, at least hard-to-understand parts
- [x] I have build all projects affected by the changes in this PR
- [x] I have tested in hardware affected projects, at the relevant boards
- [x] I have signed off all commits from this PR
- [ ] I have updated the documentation (wiki pages, ReadMe etc), if applies

### CODE REVIEW

**Comment by spectrum70** [comment]
@buha @dbogdan could you have a look to this ? thanks

File: drivers/dac/ad3552r/iio_ad3552r.c [hunk 1]
```diff
@@ -82,7 +82,7 @@ struct iio_ad3552r_desc {
 /************************* Functions Definitions *****************************/
 /*****************************************************************************/
 
-static int iio_ad3552r_attr_get(void *device, char *buf, size_t len,
+static int iio_ad3552r_attr_get(void *device, char *buf, uint32_t len,
```
Comment (buha) [inline drivers/dac/ad3552r/iio_ad3552r.c:85]: both changes in this commit are good but i'd put this type change in a separate commit where i document in the description that the type was changed to match the struct iio_attribute show and store prototypes

Comment (spectrum70) [reply drivers/dac/ad3552r/iio_ad3552r.c:85]: Thanks, done.

**Review by buha — APPROVED** [review]

**Comment by buha** [comment]
is "wip" in title still relevant ?

**Review by amiclaus — APPROVED** [review]

**Comment by amiclaus** [comment]
> is "wip" in title still relevant ?

I think it's related to the branch name.

**Comment by buha** [comment]
ok nevermind, merging then
